ocs_ci/utility/utils.py

```diff
@@ -381,6 +381,8 @@ def run_cmd(cmd, secrets=None, **kwargs):
             This kwarg is popped in order to not interfere with
             subprocess.run(**kwargs)
 
+        timeout (int): Timeout for the command, defaults to 600 seconds.
```

Comment:
This parameter is actually not in the arguments in the function so I guess we should document it under the kwargs or Keword arguments? 

Also there is one empty line above which should't be there.

Reply:
I will fix that,  **kwargs is essentially a named param but the syntactic sugar hides the param.